site/src/design/README.md

```diff
@@ -0,0 +1,7 @@
+# Armeria official logos
+
+- The logos in this folder are trademarked.
+- Read [the guideline (PDF)](armeria_guildeline.pdf) before you use the logo file.
```

Comment:
armeria_guildeline.pdf -> armeria_guideline.pdf (`l` should be removed) :-)

Reply:
Oops! Fixed :-)